types/ramda/index.d.ts

```diff
@@ -1583,6 +1584,12 @@ declare namespace R {
         /**
          * Returns the nth element in a list.
          */
+        nth<T>(n: 0, list: [T, ...any[]]): T;
+        nth<T>(n: 1, list: [any, T, ...any[]]): T;
+        nth<T>(n: 2, list: [any, any, T, ...any[]]): T;
+        nth<T>(n: 3, list: [any, any, any, T, ...any[]]): T;
+        nth<T>(n: 4, list: [any, any, any, any, T, ...any[]]): T;
+        nth<T>(n: 5, list: [any, any, any, any, any, T, ...any[]]): T;
```

Comment:
I can merge this, but wouldn't the following definition allow for an arbitrary `n`?

```ts
nth<N extends number, A extends [any?, ...any[]]>(n: N, list: A): A[N];
```

Reply:
I'll try this out